ChallengesWithTestsMarkVII/ChallengesSet01.cs

```diff
@@ -8,37 +8,75 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            return num1 == num2;
         }
 
         public double Subtract(double minuend, double subtrahend)
         {
-            throw new NotImplementedException();
+            double result = (minuend - subtrahend);
+            return result;
```

Comment:
This works, but the variable `result` is only used to store the value until the next line down.  Consider just having `return minuend - subtrahend;`